src/condor_io/condor_crypt.cpp

```diff
@@ -96,13 +104,19 @@ Condor_Crypto_State::~Condor_Crypto_State() {
 }
 
 void Condor_Crypto_State::reset() {
-    dprintf(D_SECURITY | D_VERBOSE, "CRYPTO: resetting m_ivec(len %i) and m_num\n", m_ivec_len);
+    dprintf(D_ALWAYS, "CRYPTO: protocol(%i) resetting.\n", m_keyInfo.getProtocol());
 
-    if(m_ivec) {
-	memset(m_ivec, 0, m_ivec_len);
+    if(m_keyInfo.getProtocol() == CONDOR_AESGCM) {
+        dprintf(D_SECURITY | D_VERBOSE, "ZKM: *****: CRYPTO: IGNORING AES reset.\n");
+        //Condor_Crypt_AESGCM::resetState(m_keyInfo.getConnCryptoState());
+    } else {
+        dprintf(D_SECURITY | D_VERBOSE, "CRYPTO: simple reset m_ivec(len %i) and m_num\n", m_ivec_len);
```

Comment:
Possibly too verbose.  If you want to keep it, maybe `D_NETWORK`?

Reply:
This is a dual-level debug flag.  It shows up with D_SECURITY:2 only, not on D_SECURITY.  I think it's okay.